simulation/traffic_simulator/include/traffic_simulator/entity/pedestrian_entity.hpp

```diff
@@ -34,9 +34,27 @@ namespace entity
 class PedestrianEntity : public EntityBase
 {
 public:
-  PedestrianEntity(
-    const std::string & name, const traffic_simulator_msgs::msg::PedestrianParameters & parameters,
-    const std::string & plugin_name = "behavior_tree_plugin/PedestrianBehaviorTree");
+  struct BuiltinBehavior
+  {
+    static auto behavior_tree() noexcept -> const std::string &
+    {
+      static const std::string name = "behavior_tree_plugin/PedestrianBehaviorTree";
+      return name;
+    }
+
+    static auto context_gamma() noexcept -> const std::string &
+    {
+      static const std::string name = "TODO";
+      return name;
+    }
+
+    static auto default_behavior() noexcept -> const std::string & { return behavior_tree(); }
```

Comment:
Can you follow section 4.4 and 4.5 in this coding guidelines?

http://wiki.ros.org/CppStyleGuide

Reply:
1bd92c2 fixed this.